src/layout/tablayout.js

```diff
@@ -177,20 +176,17 @@ var TabLayout = Layout.$extend({
         }
     },
 
-    //
-    setChildrenNames: function (childrenNames) {
-        this.$super(childrenNames);
-        if (!this.activeTabName) {
-            this.activeTabName = null;
-        }
-    },
-
     //////////////////////////////////////////
     // Methods                              //
     //////////////////////////////////////////
 
     // ====== Public methods ======
 
+    show() {
+        this.$super();
+        if (!this._activeTabName) this.activeTabName = null;
```

Comment:
This should not be implemented is the `show()` method as it is a shorthand for `this.setVisible(true)` (see its implementation in [the Widget class](https://github.com/wanadev/PhotonUI/blob/master/src/widget.js#L341-L343)). You must implement this in the `setVisible()` method instead.

Reply:
I made the change